valve/addons/amxmodx/scripting/agmodx.sma

```diff
@@ -362,13 +365,29 @@ public plugin_precache() {
 	// reset this always
 	gCvarMatchRunning = create_cvar("sv_ag_match_running", "0");
 	set_pcvar_string(gCvarMatchRunning, "0");
+	
+	new fwPreConfig, fwPostConfig, fwReturnTemp;
+	
+	fwPreConfig = CreateMultiForward("agmodx_pre_config", ET_IGNORE);
+	if (fwPreConfig < 0) log_amx("Error creating forward");
```

Comment:
Needs to be more explicit which forward isn't being created, or maybe remove the logging, probably when a plugin tries to use the forward an error will show up anyway, I dont remember.

Reply:
Done. I removed the logging